templates/api/clients/ruby/Gemfile.lock.tpl

```diff
@@ -1,27 +1,29 @@
 {{- $_ := stencil.ApplyTemplate "skipGrpcClient" "ruby" -}}
+{{ file.Static }}
 PATH
   remote: .
   specs:
-    {{ .Config.Name }}_client (1.63.0)
-      grpc (~> 1.38)
+    {{ .Config.Name }}_client (0.0.0)
+      grpc (~> 1.59)
 
 GEM
   remote: https://rubygems.org/
   specs:
-    google-protobuf (3.19.1)
-    googleapis-common-protos-types (1.3.0)
-      google-protobuf (~> 3.14)
-    grpc (1.42.0)
+    google-protobuf (3.25.0)
+    google-protobuf (3.25.0-arm64-darwin)
+    googleapis-common-protos-types (1.10.0)
       google-protobuf (~> 3.18)
+    grpc (1.59.2)
+      google-protobuf (~> 3.24)
       googleapis-common-protos-types (~> 1.0)
-    rake (13.0.6)
-
+    rake (13.1.0)
 PLATFORMS
   -darwin-20
+  arm64-darwin-22
```

Comment:
I'm not 100% sure if this is good or bad. Maybe @malept would know.

Reply:
So, if I remember correctly, the suffix number will change every time there's a new macOS release. For the actual function of this section, I'd need to do some research which would happen tomorrow at the earliest.